src/compose/ComposeBox.js

```diff
@@ -250,6 +263,11 @@ class ComposeBox extends PureComponent<Props, State> {
     dispatch(addToOutbox(this.getDestinationNarrow(), message));
 
     this.setMessageInputValue('');
+
+    if (this.mentionWarnings) {
+      this.mentionWarnings.getWrappedInstance().clearMentionWarnings();
```

Comment:
Hmm -- at the other spot this is `this.mentionWarnings.current.getWrappedInstance()`.

Is there a reason for the difference? I'm guessing it's unintentional.

Do they both work?

The examples in the docs at https://reactjs.org/docs/refs-and-the-dom.html seem to use `current`. Probably we should too, then, even if the other way also seems to work.